### ISSUE TITLE
Not able to save receiving addresses in desktop wallet

### ISSUE THREAD

**Issue by achterhooker** [Gulden/gulden-official#152]
Not able to save receiving addresses in the desktop wallet. 

To reproduce: 

1. Start the Gulden Desktop wallet
2. In the menu go to File > Receiving Addresses
3. Press New to create a new receiving address
4. In the popup you can enter the label field but the address field stays greyed out

### Expected behaviour
You should be able to enter and save a receiving address here. 

### Actual behaviour
You can fill out the label field, but the address field stays greyed out. Therefor you cannot enter and save an address here. The only way out is pressing the Cancel button. 

![adres](https://user-images.githubusercontent.com/46398409/51620624-12294b00-1f33-11e9-8e09-484f5b7127c2.jpg)

Gulden Desktop version 2.1.0.3

Running on Windows 64 bit

Additional info:
Saving sending addresses which works the same does work. Issue only occurs when trying to save a receiving address.

**Comment by devlz303** [comment]
I don't think you are supposed to manually add a new receiving address, because receiving addresses are part of your wallet.

However right now the receiving address book is always empty so you should be able to add them by (A.) selecting them in the new address dialog or (B.) the receiving address book should be filled with all previously used receiving addresses (but this can be a long list).

**Comment by achterhooker** [comment]
i agree. i couldn't find a reason why i would manually add a receiving address there myself but maybe some users do. If the option is there, it should work or the functionally should be clear to the end user.
